plugins/source_scheduler_dfs.go

```diff
@@ -48,7 +58,7 @@ func (p *SourcePlugin) syncDfs(ctx context.Context, spec specs.Source, client sc
 		if table.Multiplex != nil {
 			clients = table.Multiplex(client)
 		}
-		p.metrics.initWithClients(table, clients)
+		// p.metrics.initWithClients(table, clients)
```

Comment:
Isn't this already called outside the Go routine? Can you share the stack trace of the concurrency error?

I might need to look at this in the morning with fresh 👀